### ISSUE TITLE
AV: Add guidance to media placeholder image

### ISSUE THREAD

**Issue by 12** [bbc/simorgh#3139]
**Is your feature request related to a problem? Please describe.**
We need to add guidance messages to the placeholder images for media assets.

**Describe the solution you'd like**
As we have the entire `aresMedia` block to our disposal, we know if a media asset comes with guidance messages. This means that we should easily be able to add the guidance messages to our placeholder image, with our UX spec (including the screen reader UX) in mind.

![image](https://user-images.githubusercontent.com/52247014/67687625-82686f80-f990-11e9-89a2-e6dbaa25e80c.png)

**Describe alternatives you've considered**

**Testing notes**
This will require an `a11y` swarm once the logic is ready/available. This will require cross-browser testing to make sure there's no overlapping UI. We will implement Cypress tests to make sure the component appears if there is a `guidance` message.

Dev insight: Snapshots, Cypress tests to ensure the component is visible when a video block has a guidance message.

**Additional context**
Guidance/Warnings: The terminology is confusing because different systems use the opposite names for each thing. We should use what Ares calls `warnings.short` in the field SMP calls `guidance`
- SMP guidance is things like `This video contains swears`
- SMP warnings are things like `This video has no sound`
- SMP warnings are a nice to have that MPV and Ares doesn't currently support.

**Comment by lizcameron** [comment]
Updated design for Guidance message from UX: https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/dashboard?seid=5c59ad214e1897355941da14
Screenreader UX to follow.

**Comment by lizcameron** [comment]
Screenreader UX: https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/screen/5c59ad1aa48da042125184d0

**Comment by PriyaKR** [comment]
Is the guidance just been added to canonical media player page or even on amp?
